TestAppTypescript/ios/TestAppTypescript.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,1923 @@
+// !$*UTF8*$!
```

Comment:
Please apply this patch for push/codesign for our team: 

[project.pbxproj.txt](https://github.com/Microsoft/AppCenter-SDK-React-Native/files/2381312/project.pbxproj.txt)